src/agenda.rs

```diff
@@ -525,7 +536,7 @@ pub fn lang<'a>() -> Box<dyn Action> {
 
     //https://github.com/rust-lang/rfcs/pulls?q=is%3Aopen+is%3Apr+label%3AI-nominated+label%3AT-lang
     queries.push(QueryMap {
-        name: "nominated_rfcs",
+        name: "nominated_rfcs_t_lang",
```

Comment:
I wouldn't change this one because it doesn't belong to prioritization wg.

Reply:
I thought it was more intuitive to have these labels with similar names but I won't bikeshed names, so this change will be reverted